org/issue.ts

```diff
@@ -4,27 +4,91 @@ const gh = danger.github as any;
 const issue = gh.issue;
 const repo = gh.repository;
 
-const qWords: string[] = ["who", "what", "where", "when", "why", "how"]
-
-if (issue.title.slice(-1) == "?") {
-  addLabel()
-} else {
-  const title = issue.title.toLowerCase()
-  for (var i = 0; i < qWords.length; i++) {
-    if (title.startsWith(qWords[i])) {
-      addLabel()
-      break
-    }
+const title = issue.title.toLowerCase()
+const titleWords = title.split(" ")
+
+var labelsToAdd: string[] = []
+
+// label: question
+
+const questionWords: Set<string> = new Set(["how", "who", "what", "where", "when", "why", "which"])
+
+if (titleEndsInQuestionMark() || titleStartsWithAny(questionWords)) {
+  labelsToAdd.push("question")
+}
+
+// label: documentation
+
+const documentationWords: Set<string> = new Set(["documentation", "document", "docs", "doc", "readme", "changelog"])
+
+if (titleIncludesAny(documentationWords)) {
+  labelsToAdd.push("documentation")
+}
+
+// label: enhancement
+
+const enhancementWords: Set<string> = new Set(["add", "allow", "improve", "upgrade", "update", "support"])
+
+if (titleStartsWithAny(enhancementWords)) {
+  labelsToAdd.push("enhancement")
+}
+
+// label: cocoapods
+
+const cocoaPodsWords: Set<string> = new Set(["pod", "cocoapod", "cocoapods"])
+
+if (titleIncludesAny(cocoaPodsWords)) {
+  labelsToAdd.push("cocoapods")
+}
+
+// label: carthage
+
+const carthageWords: Set<string> = new Set(["carthage", "cartfile"])
+
+if (titleIncludesAny(carthageWords)) {
+  labelsToAdd.push("carthage")
+}
+
+// label: bug?
+
+const bugWords: Set<string> = new Set(["bug", "crash", "leak"])
```

Comment:
The following pattern for a bug report is really common:
`not [verb past tense || verb present particple]`
 
Personally, I don't want everything labeled `bug?` because a lot of these are not bugs but rather user errors, but it is fun to talk about 😃